inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -125,6 +125,87 @@ class XmlSerializer : public ngraph::AttributeVisitor {
         return join(adapter.get());
     }
 
+    void input_descriptions_on_adapter(const std::vector<std::shared_ptr<
+                                        ngraph::op::util::SubGraphOp::InputDescription>>& input_descriptions,
+                                        std::vector<std::string> parameter_mapping,
+                                        std::vector<std::string> result_mapping,
```

Comment:
Looks like we could pass `result_mapping` & `parameter_mapping` by const&. The same comment for other helpers.

Reply:
Sure, will do that in a second